doc/src/manual/performance-tips.md

```diff
@@ -1428,7 +1428,7 @@ julia> @code_warntype f(3.2)
 Variables
   #self#::Core.Compiler.Const(f, false)
   x::Float64
-  y::Union{Float64, Int64}
+  y::UNION{FLOAT64, INT64}
```

Comment:
I guess that by turning this all uppercase you wanted to emulate the fact that in the terminal this is printed red?
Personally I would find it more surprising to see the different capitalisation. Maybe we could actually add the color here?

Reply:
I think we also do apply this effect in the REPL if color is not supported.

Reply:
Of course, if we could render the color in the docs that'd be even better.

Reply:
> I think we also do apply this effect in the REPL if color is not supported.

Yes

Reply:
A bit later in the doc, there was capitalization, so I made the doc consistent in this regard.